molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -31,8 +31,12 @@
 
     <#if app_settings.cssHref?has_content>
         <link rel="stylesheet" href="<@resource_href "/css/${app_settings.cssHref?html}"/>" type="text/css">
-    </#if> 
-    
+    </#if>
+        <#if app_settings.customJavascript?has_content>
+            <#list app_settings.customJavascript?split(r"\s*,\s*", "r") as js_file_name>
+                <script src="<@resource_href "/js/${js_file_name?html}"/>"></script>
```

Comment:
What if I want to use JS from an external source?
e.g.: `<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js" charset="utf-8"></script>`